src/sidebar/components/tag-list.js

```diff
@@ -5,8 +5,19 @@ import propTypes from 'prop-types';
 import { isThirdPartyUser } from '../util/account-id';
 import { withServices } from '../util/service-context';
 
+/** @typedef {import('../../types/api').Annotation} Annotation */
+
+/**
+ * @typedef TagListProps
+ * @prop {Annotation} annotation - Annotation that owns the tags.
```

Comment:
Seeing this dependency is making me think we should pass in a prop indicating whether or not to render links rather than determining it locally. That's an exercise for another time.